docs/library/cn0363/cn0363_phase_data_sync/index.rst

```diff
@@ -0,0 +1,97 @@
+.. _cn0363_phase_data_sync:
+
+CN0363 Phase Data Sync FPGA Peripheral
+================================================================================
+
+.. hdl-component-diagram::
+
+The CN0363 Phase Data Sync FPGA Peripheral is part of the :ref:`EVAL-CN0363-PMDZ HDL
+reference design <cn0363>` and is responsible for preparing the ADC conversion 
```

Comment:
don't split roles between lines like this, even though sphinx allows it, it makes pattern matching harder if we need to batch patch the doc.